addons/arsenal/functions/fnc_addRightPanelButton.sqf

```diff
@@ -26,6 +26,7 @@ params [["_items", [], [[]]], ["_tooltip", "", [""]], ["_picture", QPATHTOF(data
 if (isNil QGVAR(customRightPanelButtons)) then {
     GVAR(customRightPanelButtons) = [nil, nil, nil, nil, nil, nil, nil, nil, nil, nil];
 };
+_items = _items apply {_x call EFUNC(common,getConfigName)}; // sanitize to configCase, required for cache, this doesn't run constantly anyway
```

Comment:
Move to where it's needed, as if it can't find a position it would be wasted perf.